lib/main_common.pm

```diff
@@ -88,6 +89,11 @@ sub logcurrentenv {
     }
 }
 
+sub is_tumbleweed {
+    return 1 if check_var('VERSION', 'Tumbleweed');
+    return check_var('DISTRI', 'opensuse') && get_var('VERSION') =~ /^Staging:[A-Z]/;
```

Comment:
Can you put a comment here why the staging needs to be handled as part of is_Tumbleweed?